web/client/epics/featuregrid.js

```diff
@@ -561,5 +561,19 @@ module.exports = {
                 const filter = (q || {}).filterObj;
                 const layer = (f || {}).selectedLayer;
                 return Rx.Observable.merge(Rx.Observable.of(isSyncWmsActive(store.getState())).filter(a => a), action$.ofType(START_SYNC_WMS)).map(() => addFilterToWMSLayer(layer, filter));
-            })
+            }),
+    /**
+     * addCustomEditors epic
+     * it extends the list of custom editors used in the feature grid editing
+     * and fetched by some regex rule placed in the localconfig - FeatureEditor
+     */
+    addCustomEditors: (action$) =>
```

Comment:
add test

Reply:
from what i've understood:
normally i should not use an epic for managing full javacript object, but i can use it in a custom project?